src/tests/binding.test.tsx

```diff
@@ -309,4 +309,33 @@ describe("element binding", () => {
     expect(arrow2.startBinding?.elementId).toBe(container.id);
     expect(arrow2.endBinding?.elementId).toBe(rectangle1.id);
   });
+  
+  it("should text be centered when binding to a rectangle", async () => {
```

Comment:
can you move this test to `textwysiwyg.test.tsx` ?, 
All text containers related tests we keep in the above file though can decouple later but for now lets keep it in same place so its easier to decouple later as well.
This file deals with tests related to `binding.ts`

Reply:
I don't see this file in the test folder. Should I create <code>textwysiwyg.test.tsx</code>?

Reply:
Do I really need to move this test? I notice in #6450, a comment says this test should be in <code>binding.test.tsx</code>.

Reply:
Yes please move it to https://github.com/excalidraw/excalidraw/blob/master/src/element/textWysiwyg.test.tsx, thats where we have all the tests related to text containers except for labelled arrows.

Reply:
done.